components/Weathercode.tsx

```diff
@@ -0,0 +1,33 @@
+import { Card } from "react-bootstrap";
+
+interface ComponentProps {
+    weathercode: number;
+    isDayOrNight: number;
+  }
+
+export default function Weathercode(props:ComponentProps) {
+    const {weathercode, isDayOrNight} = props;
+
+    return <>
```

Comment:
Poate aici daca poti sa tratezi putin mai elegant tot ce e sub acest return, e putin greu de urmarit, mai ales de exemplu daca ar trebui sa lucreze si altcineva in zona asta

Reply:
De exemplu, daca poti sa faci o functie care sa primeasca niste parametri si in functie de ei sa returneze ce ai tu nevoie aici